test/powershell/Language/Classes/Scripting.Classes.Break.Tests.ps1

```diff
@@ -67,8 +67,8 @@ class A
     }
 }
 '@
-            $errors.Count | Should be 1
-            $errors[0].ErrorId | Should be 'LabelNotFound'
+            $errors.Count | Should -Be 1
+            $errors[0].ErrorId | Should -Be 'LabelNotFound'
```

Comment:
Look at all instances where `Should -Be <text>` and if the text has casing, determine if it should be `Should -BeExactly` which checks for casing.